.github/workflows/rust.yml

```diff
@@ -30,6 +30,11 @@ env:
   # RUSTFLAGS: -D warnings
   RUSTDOCFLAGS: -D warnings
 
+  # Tests written with #[sim_test] are often flaky if run as #[tokio::test] - this var
+  # causes #[sim_test] to only run under the deterministic `simtest` job, and not the
+  # non-deterministic `test` job.
+  SUI_SKIP_SIMTESTS: 1
+
```

Comment:
nit: maybe it makes sense to have this env var only hang off the test job instead of apply to the whole workflow?